Demos/freeze/cython_freeze

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python
+"""
+Create a C file for embedding one or more Cython source files.
```

Comment:
I was having import issues trying to run in the source tree, but I can't reproduce that -- moved it back to ../../bin